src/interactivity/win32/window.cpp

```diff
@@ -49,7 +49,7 @@ using namespace Microsoft::Console::Render;
 ATOM Window::s_atomWindowClass = 0;
 Window* Window::s_Instance = nullptr;
 
-Window::Window() :
+Microsoft::Console::Interactivity::Win32::Window::Window() :

```

Comment:
wait why can't we `using Microsoft::Console::Interactivity::Win32` here?

Reply:
my bad. Might've been a bad merge...